tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
 envlist=
-    py{35,36}-{core,database,transactions,vm,native-blockchain}
```

Comment:
I'm still a bit confused why the `GeneralStateTests` also exists outside of the `BlockchainTest` and if there is something we might be forgetting about dropping those. Do you know why they exist in the first place? Did you check if the number of tests that are run still matches after this change?

Reply:
Upstream docs:

https://ethereum-tests.readthedocs.io/en/latest/test_types/state_tests.html
https://ethereum-tests.readthedocs.io/en/latest/test_types/blockchain_tests.html

So, you should be confused as to why `GeneralStateTests` exist inside `BlockchainTests`, not the other way around. ^_^

Anyway, according to the docs, all of the former should eventually make their way into the latter:

https://ethereum-tests.readthedocs.io/en/latest/generating-tests.html#advanced-converting-a-generalstatetest-case-into-a-blockchaintest-case